src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Common.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="12.0" DefaultTargets="PublishOutputLeg" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <RunningOnCore Condition="'$(MSBuildRuntimeType)' == 'Core'">true</RunningOnCore>
```

Comment:
I wouldn't even define this RunningOnCore property just for this one usage.

Reply:
Removed.